CONTRIBUTING.md

```diff
@@ -26,12 +26,35 @@ cp .env.dist .env
 docker-compose up
 ```
 
-Now you should be able to view the tool at [http://localhost:8042/](http://localhost:8042/)
+Now you should be able to view the tool at [http://localhost:8042/](http://localhost:8042/).
 If you need to change the port, edit `TOOLFORGE_DOCKER_PORT` in your `svgtranslate/.env` file.
 
-To run Composer commands on the server, connect to the container:
+The usual commands you'll need are as follows:
 
-    docker exec -it -u dev svgtranslate_web bash
+1. If you change anything in `assets/` you'll need to rebuild the asset files
+   (running `npm install` before the first time doing this, if you haven't already):
+
+       docker exec svgtranslate_assets npm run assets
+
+   Or, to have it monitor the files for changes while you're working
+   (note the double-hyphens to to prevent the `watch` flag from being treated as an argument to `npm`):
+
+       docker exec svgtranslate_assets npm run assets -- --watch
```

Comment:
Maybe just make this the default command so the user doesn't have to do anything? (You could override this for production)

Reply:
I experimented with that, but it got in the way of running a final build before committing some change (i.e. there are old asset files left over when `--watch` is used).